packaging/windows/WiXTools/project.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilationOptions": {
+      "emitEntryPoint": true
+  },
+
+  "dependencies": {
+    "WiX": "3.10.0.2103-pre1"
```

Comment:
You are right Peter. It does remove paths (added after installation) from %path% after uninstallation. 
But interestingly it does not remove paths added by other installations. For example
- install dotnet
- install git
- %path% has both path to dotnet and git
- uninstall dotnet
- %path% has path to git. Path to dotnet is removed.